### PR TITLE
Rename totals and reports endpoints to use `entity_type` instead of `committee_type`

### LINKED ISSUE #4886: Rename endpoint  `/totals/{committee_type}/` to  `/totals/{entity_type}/`
### What we’re after

We want to add a `committee_type` filter to the `/totals/{committee_type}/` endpoint. However, the field `committee_type` is already being used for a somewhat arbitrary entity/category grouping. To move forward, we'll need to rename the endpoint `/totals/{committee_type}/` to `/totals/{entity_type}/`
Screen Shot 2021-06-04 at 10 26 25 AM

This is **not** a breaking change because the parameter is passed through the URL - we'll just be changing the behavior under the hood and the swagger interface will have a different label.

### Related tickets

Blocking https://github.com/fecgov/openFEC/issues/4878

### Completion criteria

- [x] Rename endpoint  /totals/{committee_type}/ to  /totals/{entity_type}/
- [x] Rename `committee_type` filter to `entity_type`
- [x] Decide if we should rename `/reports/<string:committee_type>/` too :/

### PR DESCRIPTION
**Pull request by lbeaufort**
## Summary (required)

- Resolves #4886

Rename `/totals/<string:entity_type>/` and `/reports/<string:entity_type>/` endpoints to use `entity_type` instead of `committee_type`

### Required reviewers

2 developers, please

## Impacted areas of the application

-  No user impact - only under the hood.

## How to test

(Include any information that may be helpful to the reviewer(s). This might include links to sample pages to test or any local environmental setup that is unusual such as environment variable (never credentials), API version to point to, etc)

- Check out this branch
- Try queries for `/totals/<string:entity_type>/`:  http://localhost:5000/v1/totals/pac-party/?sort_nulls_last=false&sort=-committee_id&sort_null_only=false&per_page=20&sort_hide_null=false&api_key=DEMO_KEY&cycle=2022&page=1
- Try queries  for `/reports/<string:entity_type>/` http://localhost:5000/v1/reports/pac-party/?sort_nulls_last=false&sort=-committee_id&sort_null_only=false&per_page=20&sort_hide_null=false&api_key=DEMO_KEY&cycle=2022&page=1

### CODE REVIEW

File: webservices/resources/reports.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def get_match_filters():
     tags=['financial'],
     description=docs.REPORTS,
     params={
-        'committee_type': {
+        'entity_type': {
             'description': 'House, Senate, presidential, independent expenditure only',
```
Comment (fec-jli) [inline webservices/resources/reports.py:132]: Please update the description to include pac & party and add the description to docs.py.
Screen Shot 2021-06-21 at 3 25 58 PM

**Comment by fec-jli** [comment]
Good job. I and Priya review you PR, we leave two small comments there,
Please look at them.

**Comment by fec-jli** [comment]
1. Do you think we need update endpoint description to include entity_type?

2. Do we need to remove the last sentence "Parameter `full_election` is replaced by `election_full`. Please use `election_full` instead." ? 
Screen Shot 2021-06-21 at 3 27 26 PM

**Review by pkfec — APPROVED** [review]
Thanks @lbeaufort for making the changes promptly.Looks good.

**Review by fec-jli — APPROVED** [review]
good job. Thanks

**Pull request merged**